scripts/assemble.py

```diff
@@ -111,19 +112,27 @@ def main():
             help='Signed image file for secondary image')
     parser.add_argument('-o', '--output', required=True,
             help='Filename to write full image to')
-    parser.add_argument('-z', '--zephyr-base', required=True,
+    parser.add_argument('-z', '--zephyr-base',
             help='Zephyr base containg the Zephyr repository')
 
     args = parser.parse_args()
 
-    sys.path.insert(0, os.path.join(args.zephyr_base, "scripts", "dts"))
+    zephyr_base = args.zephyr_base
+    if zephyr_base is None:
+        try:
+            zephyr_base = str(os.environ['ZEPHYR_BASE'])
```

Comment:
Is the 'str' necessary here? Environment variables should always be strings. If it wasn't a string (say a byte array), this wouldn't do what you want, anyway (`str(b'foo')` results in the string `"b'foo'"`.)

Reply:
Nah, I was testing something else and forgot to remove it, thanks for pointing it out. Also fixed a typo in the help message I've missed before...